microprofile/WFLY-17138-convert-mp-metrics-subsystem-to-legacy-subsystem.adoc

```diff
@@ -0,0 +1,75 @@
+= WFLY-17138 - Convert MP Metrics Subsystem to Legacy Mode
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Red Hat has publicly stated our disagreement with the direction of MicroProfile Metrics and feel Micrometer, a de facto industry standard, provides a better metrics solution for our customers. To that end, Micrometer support will be added (via another JIRA) and, under this JIRA, MP Metrics support will be removed from standard configurations and module moved to "legacy" mode.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-17138[WFLY-17138]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-17144[WFLY-17144]
+* https://issues.redhat.com/browse/EAP7-1686[EAP7-1686]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [X] Engineering
+
+* [X] QE
+
```

Comment:
Please pick one. This sets the scope of QE involvement. Neither options means Engineering don't write or run tests.